hack/test-cmd.sh

```diff
@@ -301,8 +301,10 @@ os::cmd::expect_success 'oc logout'
 os::cmd::expect_failure_and_text 'oc get pods' '"system:anonymous" cannot list pods'
 
 # make sure we report an error if the config file we pass is not writable
-# Does not work inside of a container, determine why and reenable
-# os::cmd::expect_failure_and_text "oc login '${KUBERNETES_MASTER}' -u test -p test '--config=${templocation}/file' --insecure-skip-tls-verify" 'KUBECONFIG is set to a file that cannot be created or modified'
+templocation="/tmp/testconfig"
+touch ${templocation}
+chmod uga-w "${templocation}"
+os::cmd::expect_failure_and_text "oc login '${KUBERNETES_MASTER}' -u test -p test '--config=${templocation}' --insecure-skip-tls-verify" 'KUBECONFIG is set to a file that cannot be created or modified'
```

Comment:
@fabianofranz @smarterclayton  While this works in the container I tested this in, I am still unsure as to why it was originally failing when `$templocation` was set to `$( mktemp )`.